gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -513,6 +515,10 @@ object ConverterUtils extends Logging {
 
   // SparkSQL DateTime functions of Velox
   final val DAY_OF_MONTH = "day_of_month"
+  final val FROM_UNIXTIME = "from_unixtime"
```

Comment:
The above comment may be misleading. If both backends share the same names for these functions, we can make a correction. Thanks!